tools/cli/adminFailoverCommands.go

```diff
@@ -273,6 +200,69 @@ func getRunID(c *cli.Context) string {
 	return ""
 }
 
+func failoverStart(c *cli.Context, params *startParams) {
+	validateStartParams(params)
+
+	targetCluster := params.targetCluster
+	sourceCluster := params.sourceCluster
+	batchFailoverSize := params.batchFailoverSize
+	batchFailoverWaitTimeInSeconds := params.batchFailoverWaitTimeInSeconds
+	workflowTimeout := time.Duration(params.failoverTimeout) * time.Second
+	domains := params.domains
+	drillWaitTime := time.Duration(params.drillWaitTime) * time.Second
+
+	client := getCadenceClient(c)
+	tcCtx, cancel := newContext(c)
+	defer cancel()
+
+	options := cclient.StartWorkflowOptions{
+		ID:                           failovermanager.WorkflowID,
+		WorkflowIDReusePolicy:        cclient.WorkflowIDReusePolicyAllowDuplicate,
+		TaskList:                     failovermanager.TaskListName,
+		ExecutionStartToCloseTimeout: workflowTimeout,
+		Memo: map[string]interface{}{
+			common.MemoKeyForOperator: getOperator(),
+		},
+	}
+	if len(params.cron) > 0 {
+		options.ID = failovermanager.DrillWorkflowID
+		options.CronSchedule = params.cron
+	} else {
+		client := getCadenceClient(c)
+		tcCtx, cancel := newContext(c)
+		defer cancel()
+
+		// block managed failover if an on-going failover drill
```

Comment:
Do we need to check for the reverse situation? 

Also if we have only one workflowID then we can get rid of this check?

Reply:
Sync offline. Changed to send pause signal